internal/elasticsearch/client.go

```diff
@@ -386,3 +407,33 @@ func WaitForNumberOfHits(ctx context.Context, indexName string, query map[string
 	err := backoff.Retry(numberOfHits, exp)
 	return result, err
 }
+
+// GetSecurityApiKey waits for the elasticsearch SecurityApiKey to return the list of Api Keys.
+func GetSecurityApiKey() (APIKey, error) {
+	data := APIKey{}
+
+	r := curl.HTTPRequest{
```

Comment:
Please use the update client code from `internal/kibana/client` for making the http requests